tests/test_files/config/rules_config_invalid.py

```diff
@@ -0,0 +1,19 @@
+from cfripper.config.filter import Filter
+from cfripper.config.rule_config import RuleConfig
+from cfripper.model.enums import RuleMode
+
+RULES_CONFIG = [
+    RuleConfig(
```

Comment:
I would add a comment above `RULES_CONFIG` to make it super clear why it is an invalid config (in this case missing a key)